swarm/network/stream/delivery.go

```diff
@@ -14,60 +14,63 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-package network
+package stream
 
 import (
 	"errors"
-	"fmt"
 	"time"
 
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/p2p/discover"
+	"github.com/ethereum/go-ethereum/swarm/network"
 	"github.com/ethereum/go-ethereum/swarm/storage"
 )
 
-const retrieveRequestStream = "RETRIEVE_REQUEST"
+const (
+	swarmChunkServerStreamName = "RETRIEVE_REQUEST"
+	deliveryCap                = 32
+)
 
 type Delivery struct {
-	dbAccess *DbAccess
-	overlay  Overlay
+	db       *storage.DBAPI
```

Comment:
DBAPI should be eliminated, the whole storage package should be refactored and conform to interfaces needed for network